manifests/c/Cencit/Bas21/1.5.12/Cencit.Bas21.locale.en-US.yaml

```diff
@@ -0,0 +1,9 @@
+PackageIdentifier: "Cencit.Bas21"
+PackageVersion: "1.5.12"
+PackageLocale: "en-US"
+Publisher: "Cencit AS"
```

Comment:
```suggestion
Publisher: "Cencit as"
```
![image](https://user-images.githubusercontent.com/11600822/122379381-baa23480-cf5e-11eb-9364-12cf73c07a87.png)

Reply:
"as" is a misspelling in the installer. "AS" with capital letters is the correct form. According to the Norwegian law for company names § 2-2, a joint-stock/limited company most have the word "aksjeselskap" or the abbreviation "AS" (capital letters) in the company name.

Reply:
https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#publisher shows `Note: With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package when it has been installed. The impact is associated with winget upgrade and winget list.`